.k8s/base/ui/kustomization.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+labels:
+- pairs:
+    app.kubernetes.io/component: ui
+  includeSelectors: true
+  includeTemplates: true
+
+resources:
+- deployment.yaml
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 0

(indentation)